### PR TITLE
Custom Insert popover hide on clicking on anything on the screen fixe…

### LINKED ISSUE #750: Multiple pop-ups at once when tool buttons are clicked.
Screenshot 2021-10-13 at 12 30 05 AM

I clicked on several tool buttons such (e.g table, link etc...) and all corresponding popups occurs. However, they stay on screen until you click again on the icon that triggered the pop-up.

I expect a pop-up to close when I do one of the following:
* click the `esc` button,
* click on the page,
* click on another pop-up (to close previously opened pop-up).

This issue can be replicated on the [publiclab editor](https://publiclab.org/post) by clicking the tool buttons as you see in the image above.

### PR DESCRIPTION
**Pull request by RaviAnand111**
…s #750

fixes #750 
follow up of #793 #794 

Hey @TildaDares @NARUDESIGNS @jywarren Please have a look at this PR
Thank You

### CODE REVIEW

**Comment by RaviAnand111** [comment]
hey @NARUDESIGNS the problem still persists about the popover not opening when clicking on the button, it only appears while clicking on the `fa-tag` icon. What should I do ?
Should I ask tilda if you are not free ?

File: src/modules/PublicLab.CustomInsert.js [hunk 1]
```diff
@@ -112,5 +115,15 @@ module.exports = function CustomInsert(_module, wysiwyg) {
         });
       }
     });
+    // for the popover to disappear when clicked anywhere on the screen
+    $(':not(".woofmark-command-insert")').click((e) => {
+      if (popoverIsOpen && $('.woofmark-command-insert').children()[0] != e.target) {
+        const popoverContainer = document.querySelector('.popover');
+        const isChildElement = popoverContainer.contains(e.target);
+        if (popoverIsOpen && !e.target.classList.contains(".woofmark-command-insert") && !isChildElement) {
```
Comment (TildaDares) [inline src/modules/PublicLab.CustomInsert.js:123]:
```suggestion
        if (popoverIsOpen && !e.target.classList.contains("woofmark-command-insert") && !isChildElement) {
```

@RaviAnand111 That should fix the button not clicking issue

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: src/modules/PublicLab.CustomInsert.js (hunk 1 above)
Comment (RaviAnand111) [reply src/modules/PublicLab.CustomInsert.js]: ThankYou so much, I spent my whole day yesterday solving this bug, 
You solved it in a second, 
I am so grateful, how did you do it, I just want to be like you.
Thank You  @TildaDares @NARUDESIGNS

Comment (RaviAnand111) [reply src/modules/PublicLab.CustomInsert.js]: 😭

Comment (RaviAnand111) [reply src/modules/PublicLab.CustomInsert.js]: I rectified the error @TildaDares 
Please have a look
Thank You

**Review by TildaDares — APPROVED** [review]
Great work @RaviAnand111!

**Comment by RaviAnand111** [comment]
Thank You both @NARUDESIGNS @TildaDares

**Comment by jywarren** [comment]
Great work everyone! Thank you @RaviAnand111 !!!

**Pull request merged**